### PR TITLE
Metalamp/nft marketplace/fix beginning of time and change auction duration to end time

### PR DESCRIPTION
**Pull request by olgaklimenko**
https://trello.com/c/VOW0hf3s/30-fix-date-in-auction
https://trello.com/c/Ck2cmsaZ/35-change-duration-to-end-time-unix-timestamp-for-auctions

### CODE REVIEW

File: MetaLamp/nft-marketplace/src/Plutus/PAB/Simulation.hs [hunk 1]
```diff
@@ -314,9 +321,15 @@ instance Builtin.HasDefinitions MarketplaceContracts where
         MarketplaceUser marketplace       -> SomeBuiltin . awaitPromise $ Marketplace.userEndpoints marketplace
         MarketplaceStart           -> SomeBuiltin . awaitPromise $ Marketplace.ownerEndpoints
 
-handlers :: SimulatorEffectHandlers (Builtin MarketplaceContracts)
-handlers =
-    Simulator.mkSimulatorHandlers def def
+slotConfiguration :: POSIXTime -> SlotConfig
+slotConfiguration beginningOfTime = SlotConfig
+        { scSlotLength   = 1000
+        , scSlotZeroTime = beginningOfTime
```
Comment (performanceArtist) [inline MetaLamp/nft-marketplace/src/Plutus/PAB/Simulation.hs:330]: Why is this necessary?

Comment (olgaklimenko) [reply MetaLamp/nft-marketplace/src/Plutus/PAB/Simulation.hs:330]: Here, I set up the current time as the time when blockchain is started. In plutus code it is hardcoded as June 2020 and we see that time in our auction frontend - that is not convenient.

**Review by performanceArtist — APPROVED** [review]
